src/hb-cff-interp-common.hh

```diff
@@ -628,7 +630,9 @@ struct ParsedValues
   {
     VAL *val = values.push ();
     val->op = op;
+#ifndef HB_NDEBUG
```

Comment:
`HB_NDEBUG` is not to be used around asserts.  Just use plain assert.  From `HB_NDEBUG` description:
```
/* HB_NDEBUG disables some sanity checks that are very safe to disable and 
 * should be disabled in production systems.  If NDEBUG is defined, enable 
 * HB_NDEBUG; but if it's desirable that normal assert()s (which are very 
 * light-weight) to be enabled, then HB_DEBUG can be defined to disable 
 * the costlier checks. */ 
```
I might completely remove it.

Reply:
I am going to remove asserts entirely.